clustering/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '2.1'
+
+
+
+services:
+   master:
+      image: kartoza/geoserver:2.18.0
+      volumes:
```

Comment:
image: ene-docker.iiasa.ac.at/...:${GEOSERVER_VERSION_TAG}

Reply:
I doubt we will be using clustering but for the sake of consistency I've changed this image name aswell.